fe/fe-core/src/main/java/org/apache/doris/analysis/ShowPartitionsStmt.java

```diff
@@ -287,6 +297,10 @@ public ShowResultSetMetaData getMetaData() {
             for (String col : result.getColumnNames()) {
                 builder.addColumn(new Column(col, ScalarType.createVarchar(30)));
             }
+        } else if (catalog instanceof IcebergExternalCatalog) {
```

Comment:
need also support in P`artitionsTableValuedFunction`?

Reply:
How about support TVF in a separate PR?